.github/workflows/nia-e2e.yaml

```
@@ -21,44 +21,88 @@ jobs:
     - name: Setup Anbox Cloud
       uses: canonical/anbox-cloud-github-action@dc99b94280cfcf44a86b62aeac25ba34f14725e0
       with:
-        channel: latest/stable
+        channel: 1.28/edge/amc-connect
 
     - name: Tune Anbox Cloud
       run: |
         amc config set container.security_updates false
 
+    - name: Setup remote amc
+      env:
+        REMOTE_AMC_URL: ${{ vars.REMOTE_AMC_URL }}
+        REMOTE_CLIENT_CERT: ${{ secrets.REMOTE_CLIENT_CERT }}
+        REMOTE_CLIENT_KEY: ${{ secrets.REMOTE_CLIENT_KEY }}
+      run: |
+        set -x
+
+        url=${REMOTE_AMC_URL}
+        if [ -z "${url}" ]; then
+            ip=$(hostname -I | awk '{print $1}')
+            url="https://${ip}:8444"
```

This will take a random address if we have multiple interfaces (excluding loopback)
Why not just take the address from AMS?

If we have yq (not sure if we do here):
```suggestion
            url="$(amc config show | yq -r '.config."core.https_address"')"
```

Or, with grep:

```suggestion
            url="$(amc config show | grep -Po 'core\.https_address:\s*\K.*')"
```